diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocComments.kt

```diff
@@ -79,3 +81,10 @@ class KdocComments(private val configRules: List<RulesConfig>) : Rule("kdoc-comm
         }
     }
 }
+
+private class KdocCommentsConfiguration(config: Map<String, String>) : RuleConfiguration(config) {
+    /**
+     * Names of directories which indicate that this is path to tests. Will be checked like "src/$testAnchor" for each entry.
+     */
+    val testAnchors = config.getOrDefault("testDirs", "test").split(',')
```

Comment:
I think we should move common configuration to a special class located somewhere in `RulesConfigReader`